app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TopicsController < ApiController
+      def index
+        @topics = Topic.all
```

Comment:
Please avoid requesting code review if the pull request contains code from another open PR